test/Interop/Cxx/namespace/Inputs/namespaces.h

```diff
@@ -0,0 +1,72 @@
+namespace Space {
+namespace Ship {
+void test1() {}
```

Comment:
Could you rename the tests to say what they are testing in the name? Right now it is difficult to understand what is important about these tests, what combinations of features are covered, and which are missing.

Reply:
Yes, good idea. I found myself getting confused as well. Let me know what you think now.